imbi/automations.py

```diff
@@ -130,22 +134,21 @@ def _generate_sonar_key(project: Project) -> str:
 
     def _generate_sonar_dashboard_link(self, root_url: str,
                                        project: Project) -> str:
-        return str(yarl.URL(root_url).with_path('/dashboard').with_query({
-            'id': self._generate_sonar_key(project),
-        }))
+        if self.automation_settings['sonar']['url']:
+            root_url = yarl.URL(self.automation_settings['sonar']['url'])
```

Comment:
Since we're re-assigning root_url here, can we remove the root_url parameter? It looks unused to me.